README.md

```diff
@@ -1,6 +1,7 @@
 # Watson Developer Cloud Go SDK
 [![Build Status](https://travis-ci.org/watson-developer-cloud/go-sdk.svg?branch=master)](https://travis-ci.org/watson-developer-cloud/go-sdk)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![wdc-community.slack.com](https://wdc-slack-inviter.mybluemix.net/badge.svg)](http://wdc-slack-inviter.mybluemix.net/)
```

Comment:
Do we really want to use "bluemix.net" and not "cloud.ibm.com"?

Reply:
The url's to `cloud.ibm.com` don't work as yet